homeassistant/components/knx/climate.py

```diff
@@ -136,9 +138,11 @@ def async_add_entities_config(hass, config, async_add_entities):
     climate = xknx.devices.Climate(
         hass.data[DATA_KNX].xknx,
         name=config.get(CONF_NAME),
-        group_address_temperature=config.get(CONF_TEMPERATURE_ADDRESS),
+        group_address_temperature=config[CONF_TEMPERATURE_ADDRESS],
```

Comment:
Did you change this line by intent?

Reply:
Yes, if either the provided configuration value is required or optional and has a default value one should use this notation. At least that's what I've been told in previous reviews.

Reply:
yes, i remember ...

Reply:
Except you didn't define a default value in line 62

Reply:
It is required.